src/test/java/net/imagej/ops/topology/eulerCharacteristic/EulerCharacteristic26NFloatingTest.java

```diff
@@ -19,7 +19,7 @@
     public void testConforms() throws AssertionError {
         final Img<BitType> img = ArrayImgs.bits(3, 3);
 
-        ops.run(EulerCharacteristic26NFloating.class, img);
+        ops.topology().eulerCharacteristic26NFloating(img);
```

Comment:
Actually, since the built-in method signatures do _not_ guarantee that your exact op is called, there is some question about whether to use these sigs, or to use `run`. Maybe worth a quick discussion. This is fine for the moment though since I think the test codebase is pretty inconsistent about it right now.